lib/storage-service.js

```diff
@@ -129,6 +145,7 @@ class StorageService {
     const manager = new LeaseManager(provider, secretID);
 
     StorageService.setEvents(manager);
+    StorageService.logManagerEvent('DEBUG', 'Created LeaseManager.', manager);
```

Comment:
Do we want to log `manager` here? That will include `manager.data` which is the secret. Maybe use `StorageService.logManagerEvent` instead?